webui/react/src/pages/F_ExpList/F_ExperimentList.tsx

```diff
@@ -287,6 +289,17 @@ const F_ExperimentList: React.FC<Props> = ({ project }) => {
 
   const onContextMenuComplete = useCallback(fetchExperiments, [fetchExperiments]);
 
+  useEffect(() => {
+    (async () => {
+      try {
+        const heatMap = await getProjectNumericMetricsRange({ id: project.id });
+        setProjectHeatmap(heatMap);
+      } catch (e) {
+        handleError(e, { publicSubject: 'Unable to fetch project heatmap' });
+      }
+    })();
+  }, [project.id]);
+
```

Comment:
nit: prevent race conditions/setState after unmount issues:
```suggestion
  useEffect(() => {
    let mounted = true;
    (async () => {
      try {
        const heatMap = await getProjectNumericMetricsRange({ id: project.id });
        
        if (mounted) {
          setProjectHeatmap(heatMap);
        }
        
      } catch (e) {
        handleError(e, { publicSubject: 'Unable to fetch project heatmap' });
      }
    })();
    return () => {
      mounted = false;
    }
  }, [project.id]);

```